ci/test/cargo-test/mzcompose.py

```diff
@@ -141,6 +141,7 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
                     f"--partition=count:{partition}/{total}",
                     # Most tests don't use 100% of a CPU core, so run two tests per CPU.
                     # TODO(def-): Reenable when #19931 is fixed
+                    f"--test-threads={cpu_count // 2}",
```

Comment:
What if `cpu_count` is `1`? 🤡

Reply:
We don't have that in CI (luckily), it would just stop:
```
error: invalid value '0' for '--test-threads <THREADS>': unrecognized value for test-threads: jobs may not be 0
```